app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -572,6 +576,93 @@ export default class SampleViewV2 extends React.Component {
     return {};
   };
 
+  countReportRows = () => {
+    const { filteredReportData, reportData } = this.state;
+    let total = reportData.length;
+    let filtered = filteredReportData.length;
+    reportData.forEach(genusRow => {
+      total += genusRow.species.length;
+      filtered += genusRow.filteredSpecies.length;
+    });
+    return { total, filtered };
+  };
+
+  filteredMessage = () => {
+    const { total, filtered } = this.countReportRows();
+    return (
+      filtered != total &&
+      `${filtered} rows passing the above filters, out of ${total} total rows.`
+    );
+  };
+
+  truncatedMessage = () => {
+    const {
+      reportMetadata: { truncatedReadsCount },
+    } = this.state;
+    return (
+      truncatedReadsCount &&
+      `Overly large input was truncated to ${truncatedReadsCount} reads.`
```

Comment:
Should we say "truncated to ${} reads prior to host and quality filters"? Otherwise, I could see it being confusing, especially in combination with the `subsamplingMessage`.

Reply:
I added "Initial input" to explicitly say its the first. Referring to the filters seems a bit repetitive.
We can bring it up with other and adjust later.